docs/css/main.css

```diff
@@ -269,28 +273,29 @@ footer a {
   position: sticky;
   top: 0;
   left: 0;
-  width: 340px;
+  width: 320px;
```

Comment:
Use a CSS variable? I'm ok if the docs don't work in IE11, though we should discuss and make sure that's ok with everyone.